frontend/src/app/accountCreation/MfaPhoneVerify/MfaPhoneVerify.tsx

```diff
@@ -0,0 +1,17 @@
+import { MfaVerify } from "../MfaVerify/MfaVerify";
+
+interface Props {
+  location: { state: { phoneNumber: string } };
+}
+
+export const MfaPhoneVerify = (props: Props) => (
+  <MfaVerify
+    hint={
+      <>
+        You should receive a phone call with a security code at{" "}
+        <b>{props.location?.state?.phoneNumber || ""}</b>. It will expire in 10
```

Comment:
Same issue: seems like this prop is guaranteed. Also, it would look pretty gnarly if it defaulted to the empty string

Reply:
Yep, should be guaranteed, will remove this as well